src/ceph-volume/ceph_volume/devices/lvm/migrate.py

```diff
@@ -0,0 +1,621 @@
+from __future__ import print_function
+import argparse
+import json
+import logging
+import os
+from textwrap import dedent
+from ceph_volume.util import system, disk, arg_validators, str_to_int, merge_dict
+from ceph_volume.util.device import Device
+from ceph_volume import conf, decorators, terminal, process
+from ceph_volume.api import lvm as api
+from ceph_volume.systemd import systemctl
+
+
+logger = logging.getLogger(__name__)
+mlogger = terminal.MultiLogger(__name__)
+
+def  lvpath_to_lv(lv_path):
+    """
+    returns LV by the specified lv_path (formatted as vg_name/lv_name)
+    """
+    try:
+        vg_name, lv_name = lv_path.split('/')
+        res_lv = api.get_first_lv(filters={'lv_name': lv_name,
+                                           'vg_name': vg_name})
+    except ValueError:
+        res_lv = None
+    return res_lv
```

Comment:
I think this should live in the `api/lvm/py` module.
Alternatively this would be well suited as a arg_validator too, i.e. a simple class that implements the `__call__` method and raises `argparse.ArgumentError` if the argument is not an LV. This would allow argparse to reject non-lv arguments and safe some manual parsing and error reporting code where this method is used below.
See https://github.com/ceph/ceph/blob/master/src/ceph-volume/ceph_volume/devices/lvm/batch.py#L178 and https://github.com/ceph/ceph/blob/master/src/ceph-volume/ceph_volume/util/arg_validators.py#L47 for an example.

Reply:
moved to api/lvm.py